core/src/main/scala/com/salesforce/op/OpWorkflow.scala

```diff
@@ -149,7 +150,16 @@ class OpWorkflow(val uid: String = UID[OpWorkflow]) extends OpWorkflowCore {
           }
         val inputsChanged = blocklistRemoved.map{ f => allUpdated.find(u => u.sameOrigin(f)).getOrElse(f) }
         val oldOutput = stg.getOutput()
-        Try(stg.setInputFeatureArray(inputsChanged).setOutputFeatureName(oldOutput.name).getOutput()) match {
+        Try(stg match {
+          // For Sequence stages, we still want to keep them and remove the blocklisted inputs
+          case s @ (_ : SequenceEstimator[_, _] | _ : SequenceTransformer[_, _]) if !inputsChanged.isEmpty => {
+            s.set(s.inputFeatures, inputsChanged.map(TransientFeature(_))).setOutputFeatureName(oldOutput.name)
+            // Resetting Metadata
+            s.setMetadata(new MetadataBuilder().build())
```

Comment:
you cannot destroy the metadata on a sequence estimator like this - better call the set input and then call setMetadata(with s.getMetadata)

Reply:
That doesn't fix my issue. When setting the input a second time, it does not remove the metadata related to features that were blocklisted. This is a problem for stages that rely on column metadata E.g SanityChecker

Reply:
if the problem is with a specific stage then I would suggest that you fix the getMetadata call to that stage so that it will recreate the metadata with the new input features taken into account - doing this will break everything that relies on metadata to work for example all the model info and feature importance caluculations